content/en/docs/Installation/pre-built-binaries.md

```diff
@@ -214,3 +214,56 @@ winsw status navidrome.xml
 ```
 
 Verify that the service has started as expected by navigating to [http://localhost:4533](http://localhost:4533), by checking the Services Management Console or by checking the log file that the service wrapper created.
+
+### MacOS Installation
+
+Navidrome can be ran by simply double-clicking the binary that has been downloaded from the [release page](https://github.com/navidrome/navidrome/releases/latest) or by running it in the command line. However, that will keep a terminal window open while Navidrome is running.
+
+To have Navidrome running in the background, we can run it as a service.
+We define a service as shown below and save that in a file named `org.navidrome.plist` in the `~/Library/LaunchAgents/` folder.
+
+The example shown assumes a few things:
+
+1. The binary has been downloaded and extracted to the `/opt/Navidrome` folder.
+2. A [configuration file](https://www.navidrome.org/docs/usage/configuration-options) for Navidrome has been created and is named `navidrome.toml` in that folder.
+3. A log file for Navidrome has been created and is named `navidrome.log` in that folder.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Label</key>
+        <string>org.navidrome</string>
```

Comment:
Any reason why this is not `<string>navidrome</string>`? I think it makes more sense to use `launchtcl` with only `navidrome` instead of `org.navidrome`

Reply:
I just took a look at the existing ones I have and based myself off that.